docs/_guide/styles.md

```diff
@@ -213,13 +268,14 @@ Two best practices for working with custom elements are:
 
 See [Custom Element Best Practices](https://developers.google.com/web/fundamentals/web-components/best-practices) for more information.
 
-## Styling elements in a shadow root
+#### Styling elements in the host's shadow root {#children}
 
-Styling elements a shadow root is very straight forward: just use selectors that match your known content.
+To style elements in a shadow root, simply use standard CSS selectors.
 
 Since CSS selectors in a shadow root only apply to elements in the shadow root, you don't need to be defensive against accidentally styling other elements in the page. This means you can generally write much simpler selectors, that are easier to reason about, and faster, than without shadow DOM.
 
 Simple selectors, like `*`, tagname, id and class selectors, are fine in a shadow root because they don't match outside the root:
```

Comment:
```suggestion
Simple selectors, like `*`, tagname, id, and class selectors, are fine in a shadow root because they don't match outside the root:
```